.github/workflows/push-jar-maven.yml

```diff
@@ -0,0 +1,44 @@
+name: Upload Jar to Maven
+
+on:
+  push:
+    tags: 
+      - v*
+jobs:
+  upload-security-jar:
+    runs-on: [ubuntu-16.04]
+
+    name: Upload Jar to Maven
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        
+      - name: Configure AWS CLI
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
Curious to know: from where this access key/ secret key come?

Reply:
It is for the s3 bucket (account: openes-infra@amazon.com) that is used for staging and release of ODFE